packages/elements/src/multi-input/index.ts

```diff
@@ -26,23 +26,11 @@ const hasChanged = (newVal: unknown, oldVal: unknown): boolean => oldVal === und
 /**
  * An input control component to display a selection of pills
  *
- * @event value-changed - Fired when new value of text field is changed.
- * Property `detail.value` will be the new value.
- *
- * @event error-changed - Dispatched when error state changes.
- * Property `detail.error` is error from validation.
- *
- * @event item-added - Fired when new pill is added.
- * Property `detail.item` is new added pill.
- * Property `detail.items` is new list of all pills.
- *
- * @event item-removed - Fired when item is removed.
- * Property `detail.item` is pill that removed.
- * Property `detail.items` is new list of all pills.
- *
- * @event item-error - Fired when item that attempt to add is invalid.
- * Property `detail.item` is item with an error.
- * Property `detail.items` a current list of pills.
+ * @fires value-changed - Fired when the user added new pill or removed pill. The event is not triggered if `value` property is changed by programmatically.
```

Comment:
29 to 32: 
* align all of them with '"the" user'
* verbs for "the user" should be singular not past simple. 
* ...is changed ~~by~~ programmatically.